style.css

```diff
@@ -580,6 +590,69 @@ textarea {
   margin-left: 80px;
 }
 
+.popup_container {
+  z-index: 10000;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  background-color: rgba(124, 124, 124, 0.7);
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+}
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! :clap: 
- [ ] I suggest that you make some adjustments to the `pop-up cards`, please make them as much as similar to the figma template, kindly follow the `background blur`, `fonts` and how the `background images` are placed please see screenshots below
- `figma template`
![Screenshot from 2022-07-21 20-00-11](https://user-images.githubusercontent.com/66230306/180283585-be763068-301c-4b15-af25-ab7c05f25c42.png)

- `your template`
![Screenshot from 2022-07-21 20-03-09](https://user-images.githubusercontent.com/66230306/180284121-5e36ac17-3f2c-45c7-a50f-072dd6a46d8c.png)

Reply:
@mwanawabangona OK I will stick to the design better and fix it. Thank you very much